internal/docs/readme.go

```diff
@@ -17,27 +17,50 @@ import (
 	"github.com/elastic/elastic-package/internal/packages"
 )
 
-// ReadmeFile for the Elastic package
-const ReadmeFile = "README.md"
+// AreReadmesUpToDate function checks if all the .md readme file are up-to-date.
+func AreReadmesUpToDate() (string, string, error) {
+	packageRoot, err := packages.MustFindPackageRoot()
+	if err != nil {
+		return "", "", errors.Wrap(err, "package root not found")
+	}
 
-// IsReadmeUpToDate function checks if the README file is up-to-date.
-func IsReadmeUpToDate() (bool, error) {
-	logger.Debugf("Check if %s is up-to-date", ReadmeFile)
+	readmeFiles, err := ioutil.ReadDir(filepath.Join(packageRoot, "_dev", "build", "docs"))
+	if err != nil {
+		return "", "", nil
+	}
+
+	errNames := ""
+	notOKNames := ""
+	for _, readme := range readmeFiles {
+		filename := readme.Name()
+		ok, err := isReadmeUpToDate(filename, packageRoot)
+		if err != nil {
```

Comment:
error swallowed

Reply:
This error is passed into the `ReadmeFile` struct and returned to the upper caller function now.